lib/modules/showImages.js

```diff
@@ -830,10 +830,10 @@ const retrieveExpandoOptions = _.memoize(async (thing, { siteModule, detectResul
 		thing.isPost() && !thing.isSelfPost() &&
 		siteModule.domains.length && siteModule.attribution !== false;
 
-	const isMuted = type => ['IMAGE', 'TEXT'].includes(type);
+	const isMuted = ({ muted, type }) => muted || ['IMAGE', 'TEXT'].includes(type);
 
 	if (mediaOptions.type === 'GALLERY') {
-		mediaOptions.muted = isMuted(mediaOptions.src[0].type);
+		mediaOptions.muted = _.every(mediaOptions.src, isMuted);
```

Comment:
Why lodash instead of `Array.prototype.every`? (I think this will always be an array?)